package com.google.android.gms.internal.ads;

import android.util.Pair;
import java.util.Iterator;
import java.util.concurrent.CopyOnWriteArraySet;

/* loaded from: classes5.dex */
final class zzzx {
    final /* synthetic */ zzaah zza;
    private zzab zzb;

    /* synthetic */ zzzx(zzaah zzaahVar, zzaag zzaagVar) {
        this.zza = zzaahVar;
    }

    public final void zza(zzcd zzcdVar) {
        CopyOnWriteArraySet copyOnWriteArraySet;
        zzz zzzVar = new zzz();
        zzzVar.zzaf(zzcdVar.zzb);
        zzzVar.zzK(zzcdVar.zzc);
        zzzVar.zzaa("video/raw");
        this.zzb = zzzVar.zzag();
        copyOnWriteArraySet = this.zza.zzj;
        Iterator it = copyOnWriteArraySet.iterator();
        while (it.hasNext()) {
            ((zzaac) it.next()).zzA(this.zza, zzcdVar);
        }
    }

    public final void zzb(long j11, long j12, boolean z11) {
        zzaai zzaaiVar;
        zzaai zzaaiVar2;
        zzcx zzcxVar;
        Pair pair;
        CopyOnWriteArraySet copyOnWriteArraySet;
        if (z11) {
            zzaah zzaahVar = this.zza;
            pair = zzaahVar.zzm;
            if (pair != null) {
                copyOnWriteArraySet = zzaahVar.zzj;
                Iterator it = copyOnWriteArraySet.iterator();
                while (it.hasNext()) {
                    ((zzaac) it.next()).zzy(this.zza);
                }
            }
        }
        zzaaiVar = this.zza.zzk;
        if (zzaaiVar != null) {
            zzab zzabVar = this.zzb;
            if (zzabVar == null) {
                zzabVar = new zzz().zzag();
            }
            zzab zzabVar2 = zzabVar;
            zzaah zzaahVar2 = this.zza;
            zzaaiVar2 = zzaahVar2.zzk;
            zzcxVar = zzaahVar2.zzi;
            zzaaiVar2.zza(j12, zzcxVar.zzc(), zzabVar2, null);
        }
        zzcw.zzb(null);
        throw null;
    }
}
